.github/workflows/deploy-tests.yml

```diff
@@ -37,14 +39,25 @@ jobs:
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       S3_GIN_BUCKET: ${{ secrets.S3_GIN_BUCKET }}
       CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
-
+    with:
+      python-versions: >  # Ternary operator,  condition && value_if_true || value_if_false
+        ${{github.event.pull_request.draft == true
+           && '["3.9"]'
```

Comment:
My thinking here is that most of us do not use the oldest version of Python.  This prevents us from getting errors because we use things not supported in Python 3.9 when we develop.